org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/workspace/internal/DefaultGradleBuilds.java

```diff
@@ -42,12 +41,8 @@ public DefaultGradleBuilds(Set<BuildConfiguration> buildConfigs) {
 
     @Override
     public void synchronize(NewProjectHandler newProjectHandler, CancellationTokenSource tokenSource, IProgressMonitor monitor) throws CoreException {
-        SynchronizeGradleBuildsOperation syncOperation = SynchronizeGradleBuildsOperation.forMultipleGradleBuilds(this, newProjectHandler);
```

Comment:
⭕️ This class seems unused now.

Reply:
I removed the method and simplified the class accordingly.